lib/connection.js

```diff
@@ -29,17 +29,16 @@ function Connection(options) {
   this.apiCallCount = 0;
 
   // Convenient methods, matches that of restler's
-  _.each(restlerMethodArgCount, function(count, method) {
+  _.forEach(restlerMethodArgCount, (count, method) => {
     this[method] = function(var_args) {
       var args = Array.prototype.slice.call(arguments);
       while (args.length < count) {
         args.push(undefined);
       }
       args.unshift(method);
-
       return this._request.apply(this, args);
     };
-  }, this);
```

Comment:
@hac-madkudu the `, this` in the previous version of `lodash.each` was to set the context of the iteratee. You get the same effect with an arrow function. 
Calling `this[method]` seemed unnecessary and likely to have side effects. Same for the returning of this?